hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/HederaEvmWorldStateTokenAccount.java

```diff
@@ -46,7 +46,7 @@ public HederaEvmWorldStateTokenAccount(final Address address) {
 
     @Override
     public Bytes getCode() {
-        return interpolatedCode().getContainerBytes();
+        return interpolatedCode().getBytes();
```

Comment:
Q: why the change? Should probably be in PR description details

Reply:
The API provided by Besu changed.